package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -426,7 +426,7 @@ static int read_charmm_4dim(fio_fd fd, int charmm, int reverseEndian) {
 
 /* XXX This is completely broken for fixed coordinates */
 static int read_dcdsubset(fio_fd fd, int N, int lowerb, int upperb, float *X, float *Y, float *Z,
-			  float *unitcell, int num_fixed, int first, int *indexes, float *fixedcoords,
+			  double *unitcell, int num_fixed, int first, int *indexes, float *fixedcoords,
```

Comment:
`unitcell` are doubles, really? (Sorry, haven't looked at the code in detail but strikes me as strange.)